keps/sig-node/3673-kubelet-parallel-image-pull-limit/README.md

```diff
@@ -374,7 +381,9 @@ https://storage.googleapis.com/k8s-triage/index.html
 We expect no non-infra related flakes in the last month as a GA graduation criteria.
 -->
 
-A new node_e2e test with  `serialize-image-pulls==false` will be added to make sure that when maxParallelImagePulls is reached, all further image pulls will be blocked.
+A new node_e2e test with  `serialize-image-pulls==false` will be added test parallel image pull limits. 
+1. When maxParallelImagePulls is reached, all further image pulls will be blocked. 
+2. Verify the behavior when the same image is pulled in parallel, which will happen when image pull policy is `Always`.
```

Comment:
Is there logic to wait requests for an image pull of a pull already in progress?

Reply:
Not sure if I understand the question, but my plan here is to verify the behavior based on the timestamp of image pulling events.

So for the first case, we will check if there are _N_ pulling events started roughly at the same time.

for the second case, we will check if there are two pulling events for the same image.